### PR TITLE
Registration page

### PR DESCRIPTION
**Pull request by kivenalaric**
registration page done

### CODE REVIEW

**Review by Nsairun — DISMISSED** [review]

**Review by kadjialan — DISMISSED** [review]

File: src/Pages/Registration/Registration.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        <Label>UserName</Label>
+        <InputField placeholder="username" name="username" value="username" />
+        <Label>Email</Label>
+        <InputField placeholder="emailAdress" name="email" value="email" />
+        <PassConfirm>
+          <PassHolder>
+            <Label>Password</Label>
+            <InputField
+              placeholder="password"
+              name="password"
```
Comment (RashJrEdmund) [inline src/Pages/Registration/Registration.jsx]: input fields with values become read only type... remove for better UX

**Review by RashJrEdmund — CHANGES_REQUESTED** [review]

File: src/Pages/Registration/Registration.jsx (hunk 1 above)
Comment (kivenalaric) [reply src/Pages/Registration/Registration.jsx]: done sir

**Review by RashJrEdmund — APPROVED** [review]

**Review by RashJrEdmund — APPROVED** [review]

**Review by RashJrEdmund — APPROVED** [review]

File: src/Pages/Registration/Registration.jsx (hunk 1 above)
Comment (RashJrEdmund) [reply src/Pages/Registration/Registration.jsx]: good good good

**Review by RashJrEdmund — APPROVED** [review]

**Review by RashJrEdmund — APPROVED** [review]

File: src/Pages/Registration/Registration.jsx (hunk 1 above)
Comment (kivenalaric) [reply src/Pages/Registration/Registration.jsx]: okay boss

**Review by RashJrEdmund — APPROVED** [review]

**Review by RashJrEdmund — APPROVED** [review]

**Review by RashJrEdmund — APPROVED** [review]

**Review by Nsairun — APPROVED** [review]

**Pull request merged**